tutorial/markdown/nodejs/quickstart-nextjs/quickstart-nextjs.md

```diff
@@ -20,6 +20,7 @@ sdk_language:
 length: 45 Mins
 ---
 
+<!-- TODO: update this to be sure its working -->
```

Comment:
Gitpod seems to have stopped working for this tutorial (and perhaps others). We need to determine how to fix this, or remove it.

Reply:
![Screen Shot 2023-07-10 at 5 07 07 PM](https://github.com/couchbase-examples/couchbase-tutorials/assets/4667097/ff6f52ac-349c-4d60-997b-c023457d2e35)